### PR TITLE
Update cop_pse_auxiliary.R（F - FALSE）

### PR DESCRIPTION
**Pull request by wwang93**

### CODE REVIEW

**Comment by jrosen48** [comment]
Looks good! Will merge later once all PRs related to #50 are completed.

File: R/cop_pse_auxiliary.R [hunk 1]
```diff
@@ -66,7 +66,7 @@ cal_delta_star <- function(FR2max,
                            est_uncond, 
                            rxz, 
                            n_obs) {
-    if (FR2max > .99) {FR2max <- .99}
+    if (FR2max > .99) {FR2max =.99}
```
Comment (jrosen48) [inline R/cop_pse_auxiliary.R]: did this possibly break the tests/R CMD check?

**Review by jrosen48 — DISMISSED** [review]
this commit seems to be associated with tests failing

File: R/cop_pse_auxiliary.R (hunk 1 above)
Comment (wwang93) [reply R/cop_pse_auxiliary.R]: It is what I wanna talk with you ,when I change the = to "<-", there is a tip to remind me this line can't go through the  test.

Comment (wwang93) [reply R/cop_pse_auxiliary.R]: but when you see [e16bc75], I changed the "=" back to "<-",still show the same issue. So I don't think the line 69 break the tests.

**Comment by wwang93** [comment]
The wwang93-addressing-good-practice-issues Branch  solves goodpractice's : "<-", "TF", and "importing function from other issues", and on the issue of long lines, we went from #50's 447, down to #78's 283 lines, mostly lengthy equations, and others are comments.

**Comment by wwang93** [comment]
wwang93-addressing-good-practice-issues to solve the goodpractice's "<-", "TF", and "importing function from other issues", and on the issue of long lines, we went from #50's 447, down to #78's 283 lines, mostly lengthy equations, and others are comments.

**Comment by jrosen48** [comment]
Noting that this addresses #50. Will comment there with an update on the results of `goodpractice::gp()`.

**Pull request merged**